Makefile

```diff
@@ -55,3 +55,6 @@ module: bin/viamrtsp
 
 clean:
 	rm -rf FFmpeg bin/viamrtsp module.tar.gz
+
+clean-all:
```

Comment:
should we combine this with clean?

Reply:
I think `clean-all` should be separate as it will delete anything that is not in version control, so it could easily lead to loss of work if done mistakenly. I think `clean` should clean the things that we expect people to want to clean frequently.

I've also updated the makefile to separate out build directories for each version of ffmpeg (currently only n6.1) and platform so that we should never need to clean the FFmpeg build directory when we switch platforms.

Reply:
Sounds good.